### PR TITLE
Add GPIO control buttons to turn on and off the lights

### PR DESCRIPTION
**Pull request by GLDuval**
This PR adds a new config menu to control our lights. These are not customizable directly in the UI for now, this feature can be added in another PR. 
image

### CODE REVIEW

File: src/renderer/store/modules/gpioPins.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { GlobalState } from '@/renderer/store/store';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { nanoid } from 'nanoid';
+
+export interface GpioPinsState {
```
[the remark this answers is not in the document]
Comment (GLDuval) [reply src/renderer/store/modules/gpioPins.ts]: You're right I just fixed it

**Review by saxtot — APPROVED** [review]
10/10, good job.

File: src/renderer/components/common/Button.tsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ const StyledButton = styled.button<StyledButtonProps>`
     ${({ theme }) => theme.colors.background},
     ${({ theme }) => theme.colors.darkerBackground}
   );
+  border-radius: 4px;
```
Comment (Leuchak) [inline src/renderer/components/common/Button.tsx:58]: It doesn't seem to affect the buttons used in GPIO control. Is it an oversight?

**Review by Leuchak — APPROVED** [review]
it looks good.

**Pull request merged**